ec2api/api/cloud.py

```diff
@@ -384,9 +384,9 @@ def revoke_security_group_egress(self, context, group_id,
                             'ip', 'str64',
                             'dummy', 'dummy',
                             'bool')
-    def run_instances(self, context, image_id, min_count, max_count,
+    def run_instances(self, context, image_id, instance_count,
                       key_name=None, security_group_id=None,
-                      security_group=None, user_data=None, instance_type=None,
+                      security_group=None, user_data=None, instance_type_id=None,
                       placement=None, kernel_id=None, ramdisk_id=None,
```

Comment:
Would it make sense to drop a few parameters from run_instances which we are definitely not going to support for now? Like placement, kernel_id and ramdisk_id.